.github/workflows/build.yml

```diff
@@ -8,11 +8,31 @@ on:
     paths-ignore:
       - 'build.gradle'
       - 'bumpver.toml'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      venue:
+        type: choice
+        description: Venue to deploy to
+        options:
+          - SIT
+          - UAT
+
 jobs:
-  build-api:
+  build:
     name: build, lint, and test API
     runs-on: ubuntu-latest
+    outputs:
+      deploy_env: ${{ steps.gradle-build.outputs.deploy_env }}
     steps:
+      - name: No version bump
+        # If triggered by workflow dispatch, no version bump
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |
+          echo "TARGET_ENV_UPPERCASE=${{ github.event.inputs.venue }}" >> $GITHUB_ENV
```

Comment:
Since we only have one `TARGET_ENV` variable, can we just call it `TARGET_ENV` without the UPPERCASE suffix?